lib/network/modules/components/shared/Label.js

```diff
@@ -7,14 +7,13 @@ let ComponentUtil = require('./ComponentUtil').default;
  *
  * This has been moved away from the label processing code for better undestanding upon reading.
  *
- * @class LabelAccumulator
+ * @callback measureText
+ * @param {function} measureText - callback to determine text dimensions, using the parent label settings.
```

Comment:
Look up the `@callback` tag in the [reference](http://usejsdoc.org/tags-param.html).

Reply:
ah, I see, I misunderstood.